sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -3233,7 +3265,7 @@ private void SetNameFieldsIfNull()
         /// The <see cref="StartCopyFromUri(Uri, BlobCopyFromUriOptions, CancellationToken)"/>
         /// operation begins an asynchronous copy of the data from the <paramref name="source"/> to this blob.
         /// You can check the <see cref="BlobProperties.CopyStatus"/>
-        /// returned from the <see cref="GetProperties"/> to determine if the
+        /// returned from the <see cref="GetProperties(BlobRequestConditions, CancellationToken)"/> to determine if the
```

Comment:
(My comment may not apply if we're getting rid of the BlobGetPropertiesOptions option bag)

I think you mean to put "GetProperties(BlobGetPropertiesOptions, CancellationToken)" or "GetProperties(BlobRequestConditions, Cancellation)" here. Since I saw in the api file that you made this method editor browsable never here.

Same for the rest below